github-dark.css

```diff
@@ -3446,7 +3446,8 @@
   }
   /* yellow text */
   /* .bg-pending might be a GitHub bug as it sets the fg */
-  .text-pending, .text-renamed, .bg-pending {
+  .text-pending, .text-renamed, .bg-pending, .page-new-repo .octicon-lock,
+  .private a svg {
```

Comment:
I don't think this selector is needed (it's for the lock on the dashboard right?). If so it should be covered by `.repo-private-icon` in https://github.com/StylishThemes/GitHub-Dark/blob/68bcb8d7cea5166b35035b5f77edf7500180610d/github-dark.css#L1155-L1159

Reply:
See https://github.com/StylishThemes/GitHub-Dark/pull/827/commits/3aa34493eeca90e0deb927148244b9cb605b1de6 removed.

Anything else?

Reply:
@xt0rted anything else?